R/activate.R

```diff
@@ -142,7 +142,7 @@ renv_activate_prompt <- function(action, library, prompt, project) {
     prompt &&
     interactive() &&
     is.null(library) &&
-    !identical(project, Sys.getenv("RENV_PROJECT"))
+    !renv_project_is_active(project)
```

Comment:
```suggestion
    !renv_project_loaded(project)
```

Reply:
Are you suggesting a different name for this function?

Reply:
Yeah, that's my intent here.